HW_7/task_7_1/task_7_1.py

```diff
@@ -0,0 +1,26 @@
+# ✔ Напишите функцию, которая заполняет файл
+# (добавляет в конец) случайными парами чисел.
+# ✔ Первое число int, второе - float разделены вертикальной чертой.
+# ✔ Минимальное число - -1000, максимальное - +1000.
+# ✔ Количество строк и имя файла передаются как аргументы функции.
+#
+from pathlib import Path
+from random import uniform, randint
+
+Min = -1000
+Max = 1000
+
+
+def fill_file(name, quanty, dir,):
+    Path(dir).mkdir(parents=True, exist_ok=True)
+    path_to_files = str(Path(dir) / name)
+    with open(f'{path_to_files}', 'a+', encoding='utf-8') as f:
+        for _ in range(quanty):
+            print(f'{str(randint(Min, Max))}|{str(uniform(Min, Max))}', file=f)
```

Comment:
Если используете форматирование, то переводить к строковому типу не нужно
```suggestion
            print(f'{randint(Min, Max)}|{uniform(Min, Max)}', file=f)
```